### ISSUE TITLE
Support uploading files directly

### ISSUE THREAD

**Issue by nadrees** [gotdibbs/wrp#1]
Ideally if we're scripting against this library I'd like to be able to specify an array of files to upload instead of a directory. Can we add a switch for this?

**Comment by gotdibbs** [comment]
Yeah, that's on my list, just need to be able to figure out a safe way of passing those files and their path fragments through to the powershell script under the hood is all.

Thanks!
